src/core/Lockfile_xtarget.ml

```diff
@@ -0,0 +1,67 @@
+type t = {
+  target : Target.lockfile;
+  manifest : manifest option;
+  lazy_content : string lazy_t;
+      (** The contents of the lockfile, as a string. *)
+  lazy_ast_and_errors : Dependency.t list lazy_t;
```

Comment:
Maybe rename this field since this does not really contain ast and errors right?

Reply:
Done in d57830ad536d2800a7e4c53159e6c156191eae83.